### ISSUE TITLE
Can not log-in with plex.tv Account

### ISSUE THREAD

**Issue by EarthlingX** [luksus42/plex-app#8]
In first: Thank you very much for your great work! Very glad to see there is a Plex app available for Ubuntu Touch! :-)

Unfortunately, within the app I can not log in with my plex.tv-Account. I pasted username and password, afterwards click on "Save". But there is always this message "Your login data seems to be incorrect!".

The same login-data works fine in Morph Browser. Any ideas about that?

**Comment by luksus42** [comment]
Hi,
honestly I don't use the anymore, because I switched to emby some time ago...
But I did a quick check with my plex account and the login process still works.

I just could imagine that you have some very special characters in your username or password. Sometimes this causes encoding errors...

Perhaps you could download the web app on your PC, extract it and execute it in a browser, then watch the console output of the browsers developer tools...


On Fri, 22 May 2020 15:16:14 -0700, EarthlingX wrote:
> In first: Thank you very much for your great work! Very glad to 
> see there is a Plex app available for Ubuntu Touch! :-)
>
> Unfortunately, within the app I can not log in with my 
> plex.tv-Account. I pasted username and password, afterwards 
> click on "Save". But there is always this message "Your login 
> data seems to be incorrect!".
>
> The same login-data works fine in Morph Browser. Any ideas about that?
>

-- 
Sent using Dekko from my Ubuntu device

**Comment by EarthlingX** [comment]
Thank you very much for your kind reply! :+1: 

> I just could imagine that you have some very special characters in your username or password. Sometimes this causes encoding errors...

Indeed, this could be true! I have some `), (, ", §, %` and `=` in my password. :-) Should I avoid these and better change the password?

**Comment by luksus42** [comment]
usually it should not be necessary to avoid special characters.
But if you don't mind, you could  do a test with a simpler password, just to determine, if my assumption is right.

In this case, I think, there should be a way to fix this.


On Sun, 24 May 2020 06:30:03 -0700, EarthlingX wrote:
> Thank you very much for your kind reply! :+1: 
> 
>> I just could imagine that you have some very special 
>> characters in your username or password. Sometimes this causes 
>> encoding errors...
> 
> Indeed, this could be true! I have some `), (, ", §, %` and `=` 
> in my password. :-) Should I avoid these?
> 
> 


--

**Comment by EarthlingX** [comment]
Great, I have changed my password and now I can log in successfully. :+1: Thanks for that hint!

However, after selecting the desired device, nothing is displayed in the screen. Did I do something wrong?

![screenshot](https://user-images.githubusercontent.com/38689554/82763397-e6ec8f80-9df6-11ea-97d4-bedb863bc202.png)

**Comment by luksus42** [comment]
Mh, as I said, I did not use the app for a while, therefor I also did not maintain it.
Maybe meanwhile something changed in the plex api.

I would have to setup a plexserver first to investigate this issue any further...
This will take some time.
You could take a look at the log output with the logviewer app. Perhaps there is some useful information in it.

Keep in mind, that this app will not show any "pictures" library. Only video and music libraries will show up, as far as I remember.
Also, the plex server should allow http requests (insecure requests, http without "s").

**Comment by EarthlingX** [comment]
Let me try that! Which logviewer app do you think of?

**Comment by luksus42** [comment]
As far as I know, there is only one "logviewer" app in the openstore.

**Comment by luksus42** [comment]
@EarthlingX : Hi again,

I just tried the app with a minimal plex server, only a small music mediathek, and everything seems fine.
I am not sure, where to go from here, as I don't get any error to work with...

**Comment by EarthlingX** [comment]
Did you try it on tablet or on phone?

So in your case, you do not see this grey screen that I always receive (see above)?

**Comment by luksus42** [comment]
Yes, there is a "folder" which represents my music library. And I can browse through it and listen to a song.

Can you please tell me, what kind of media libraries you created in your plex server ?

On Fri, 12 Jun 2020 10:13:17 -0700, EarthlingX wrote:
> Did you try it on tablet or on phone?
>
> So in your case, you do not see this grey screen that I always 
> receive (see above)?
>

--

**Comment by EarthlingX** [comment]
Thank you for your reply!

> Can you please tell me, what kind of media libraries you created in your plex server ?

Actually I am not using an own Plex server, but the shared library of the Plex server of a friend of mine:

![test](https://user-images.githubusercontent.com/38689554/84563895-6ca49080-ad4e-11ea-9481-c977ba7419e4.png)

**Comment by luksus42** [comment]
Ahh I see, I fear that this kind of shared plexserver usage is not supported by my app yet.
Perhaps I can have a look into it and make it work, if it is not too complicated.



On Sat, 13 Jun 2020 01:19:32 -0700, EarthlingX wrote:
> Thank you for your reply!
>
>> Can you please tell me, what kind of media libraries you 
>> created in your plex server ?
>
> Actually I am not using an own Plex server, but the shared 
> library of the Plex server of a friend of mine:
>
> ![test](https://user-images.githubusercontent.com/38689554/84563895-6ca49080-ad4e-11ea-9481-c977ba7419e4.png)
>

-- 


--

**Comment by EarthlingX** [comment]
> Ahh I see, I fear that this kind of shared plexserver usage is not supported by my app yet. Perhaps I can have a look into it and make it work, if it is not too complicated.

That would be really great! Please let me know if I can send you a financial aid. :-)
